### ISSUE TITLE
Kactus failed to launch on 0.3.7

### ISSUE THREAD

**Issue by synthresin** [kactus-io/kactus#105]
## Description

Kactus fails to launch

## Version

* Kactus: 0.3.7
* Operating system: 10.13.4

## Steps to Reproduce
1. Open Kactus

### Expected Behavior
Kactus should launch.

### Actual Behavior
Kactus fails to launch

## Additional Information
```
Invariant Violation: Minified React error #200; visit https://reactjs.org/docs/error-decoder.html?invariant=200 for the full message or use the non-minified dev environment for full errors and additional helpful warnings. 
    at e.exports (/webpack:/renderer.js:10:1)
    at o (/webpack:/renderer.js:66:1)
    at Ir (/webpack:/renderer.js:66:1)
    at Object.render (/webpack:/renderer.js:66:1)
    at Object.<anonymous> (/webpack:/renderer.js:73:1)
    at t (/webpack:/renderer.js:1:1)
    at /webpack:/renderer.js:1:1
    at /webpack:/renderer.js:1:1

Version: 0.3.7
OS: Mac OS 10.13.4
```

### Logs
[2018-06-12.kactus.production.log](https://github.com/kactus-io/kactus/files/2092166/2018-06-12.kactus.production.log)

**Comment by synthresin** [comment]
It happens on my co-worker's mac.

**Comment by synthresin** [comment]
Resolved on 0.3.8. Thanks.
